src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -173,6 +175,7 @@ public MailConfig(MailConfig other) {
     keepAliveTimeoutUnit = other.keepAliveTimeoutUnit;
     ntDomain = other.ntDomain;
     workstation = other.workstation;
+    countsPerConn = other.countsPerConn;
```

Comment:
rename it `maxInflightMailsPerConnection`

Reply:
Done :)